### DISCUSSION TITLE
Angular recipe for create-tauri-app?

### DISCUSSION THREAD

**Discussion by boan-anbo** [tauri-apps/tauri#5102] [Q&A] [answered]
I remember having used `create-tauri-app` to generate Angular intergrated Tauri apps, is it gone? What's the alternative now? Thanks!
![image](https://user-images.githubusercontent.com/7269717/187548585-d2674b68-28df-4549-bcfc-ac1252c53c65.png)

**Comment by amrbashir** [comment, marked as answer]
create-tauri-app has been rewritten from scratch. Until someone contributes a new angular template, your only solution is to create the angular project first then add Tauri to it.

**Reply by boan-anbo** [reply]
Got it! Thanks!
